amid/rsna_bc/utils.py

```diff
@@ -0,0 +1,51 @@
+import contextlib
+import zipfile
+from pathlib import Path
+
+import pandas as pd
+
+
+fields = {
+    'site_id': str,
+    'patient_id': str,
+    'image_id': str,
+    'laterality': None,
+    'view': None,
+    'age': None,
+    'cancer': bool,
+    'biopsy': bool,
+    'invasive': bool,
+    'BIRADS': int,
+    'implant': bool,
+    'density': None,
+    'machine_id': str,
+    'prediction_id': str,
+    'difficult_negative_case': bool,
+}
+
+
+def add_csv_fields(scope):
```

Comment:
maybe later we should make a common method for converting csv to class methods

Reply:
+1